docs/guides/external-mgs/readme.md

```diff
@@ -0,0 +1,151 @@
+---
+authors: Matt Richerson <matthew.richerson@hpe.com>
+categories: provisioning
+---
+
+# Lustre External MGS
+
+## Background
+
+Lustre has a limitation where only a single MGT can be mounted on a node at a single time. In the NNF cluster, this means the total number of Lustre file systems is limited to the number of NNF nodes. It also places extra scheduling constraints on the WLM to ensure that the MGTs don't conflict.
```

Comment:
"In the NNF cluster..."  now you're describing a problem, as if you're writing a design document.

This should talk about features rather than problems.